### ISSUE TITLE
BIG SKY DESIGN'S LATEST CREATIONS SEE ALL

### ISSUE THREAD

**Issue by cbruch** [Kducey1/BigSkyInternetDesign#16]
Let's remove the text spots & just put in 3 sites. The see all will have to be done by Tom or Jake.

**Comment by Kducey1** [comment]
I removed the text and JT was questioning the "see all" part. I would talk to him on that one

**Comment by JT-Bruch** [comment]
@tbruch just out of curiosity - the portfolio page on BSID I know its an ASP page, but is there a way to query that data via an AJAX call?

**Comment by tbruch** [comment]
I am sure there is, but I would not know how to do it.

**Comment by JT-Bruch** [comment]
Yeah, I'm thinking it might be easier to write an AJAX and inject that into the portfolio instead of making the whole page .asp.  

Also would look better for those who are technically inclined.  Anyways just a thought.
